src/linker/Linker.Steps/SweepStep.cs

```diff
@@ -472,7 +472,13 @@ protected virtual void SweepMethods (Collection<MethodDefinition> methods)
 		void SweepOverrides (MethodDefinition method)
 		{
 			for (int i = 0; i < method.Overrides.Count;) {
```

Comment:
Is this overrides of the target method? Or methods the target overrides?

Reply:
This is the methods the target overrides.

Reply:
To show it on an example:
* `method` is typically `ImplementationClass.StaticInterfaceMethod`
* `override` is typically `IStaticInterface.StaticInterfaceMethod`.